app/models/form/subsection.rb

```diff
@@ -32,7 +32,7 @@ def status(case_log)
 
     qs = applicable_questions(case_log)
     return :not_started if qs.all? { |question| case_log[question.id].blank? || question.read_only? }
-    return :completed if qs.all? { |question| question.completed?(case_log) }
+    return :completed if qs.all? { |question| question.type == "checkbox" ? question.answer_options.keys.any? { |key| case_log[key] == "Yes" } : question.completed?(case_log) }
```

Comment:
I think this logic should go into the `question.completed?` method. i.e. I should be able to ask a question whether it's completed or not without having to care what type it is.

Reply:
That's actually a more elegant solution so will get that sorted 🙂